amf-aml/shared/src/main/scala/amf/aml/internal/parse/instances/parser/ObjectMapPropertyParser.scala

```diff
@@ -65,6 +67,8 @@ object ObjectMapPropertyParser extends NodeMappableHelper {
           node.set(Field(Str, ValueType(propId)),
                    AmfScalar(propertyEntry.key.as[YScalar].text),
                    Annotations(propertyEntry.key))
+          node.annotations.reject(_.isInstanceOf[LexicalInformation]) ++= Annotations(propertyEntry)
```

Comment:
Why reject annotations ?

Reply:
because if a maptermkey is found we set the value, but the lexical was already set before, without including it, so we remove it and set a new one.

Reply:
This is still weird, why can't it be right on the first try?